app/(main)/MarketsHighlight.tsx

```diff
@@ -14,28 +13,15 @@ import {
   CarouselSelector,
 } from '../components/Carousel';
 import Autoplay from 'embla-carousel-autoplay';
-import { highlightedMarketsList } from '@/market-highlight.config';
-import defaultHighlightImage from '@/public/assets/highlights/default.png';
-import Image from 'next/image';
-import { useState } from 'react';
+import { FA_EVENTS } from '@/analytics';
+import { trackEvent } from 'fathom-client';
 
-export const MarketsHighlight = () => {
-  const { data: markets, isLoading } = useQueries({
-    queries: Object.keys(highlightedMarketsList).map(id => ({
-      queryKey: ['getMarket', id],
-      queryFn: async () => getMarket({ id }),
-    })),
-    combine: results => {
-      return {
-        data: results
-          .map(result => result.data?.fixedProductMarketMaker)
-          .filter((market): market is FixedProductMarketMaker => !!market),
-        isLoading: results.some(result => result.isPending),
-      };
-    },
-  });
+interface MarketsHighlightProps {
+  markets: any[];
```

Comment:
Is there a way we could type this or it gets too complex?

Reply:
We can use `FixedProductMarketMaker[]` here

Reply:
fixed